doc-content/shared-kie-docs/src/main/asciidoc/KieServer/controller-rest-api-templates-ref.adoc

```diff
@@ -0,0 +1,881 @@
+    }
+  ],
+  "capabilities": [
+    "RULE",
+    "PROCESS",
+    "PLANNING"
+  ]
+}
+----
+
+== [PUT] /management/servers/{serverId}
+
+Creates a new {KIE_SERVER} template with a specified ID.
+
+.Request parameters
+[cols="25%,45%,15%,15%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`serverId`
+|ID of the new {KIE_SERVER} template
+|String
+|Required
+
+|*body*
+|A map containing the `server-name`, `capabilities`, and other components of the new {KIE_SERVER} template
+|Request body
+|Required
+|===
+
+.Example request body (JSON)
+[source,json]
+----
+{
+  "server-id": "new-kieserver",
+  "server-name": "new-kieserver",
+  "container-specs": [],
+  "server-config": {},
+  "capabilities": [
+    "RULE",
+    "PROCESS",
+    "PLANNING"
+  ]
+}
+----
+
+*<@Cristiano: The request works but the server responds "undocumented". Need to update the server so that it responds with the following, or similar.>*
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "Server template new-kieserver successfully created."
+    }
+  ]
+}
+----
+
+== [DELETE] /management/servers/{serverId}
+
+Deletes a specified {KIE_SERVER} template.
+
+.Request parameters
+[cols="25%,45%,15%,15%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`serverId`
+|ID of the {KIE_SERVER} template to be deleted
+|String
+|Required
+|===
+
+*<@Cristiano: The request works but the server responds "undocumented". Need to update the server so that it responds with the following, or similar.>*
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "Server template new-kieserver successfully deleted."
+    }
+  ]
+}
+----
+
+////
+= KIE containers
+
+The {CONTROLLER} REST API supports the following endpoints for managing KIE containers (deployment units) in {KIE_SERVER} templates (configurations) and in {KIE_SERVER} instances (remote servers). The {CONTROLLER} REST API base URL is `\http://SERVER:PORT/CONTROLLER/rest/controller/`. All requests require basic HTTP Authentication or token-based authentication for the `rest-all` user role if you installed {CENTRAL} and you want to use the built-in {CONTROLLER}, or the `kie-server` user role if you installed the {HEADLESS_CONTROLLER} separately from {CENTRAL}.
+////
+
+== [GET] /management/servers/{serverId}/containers
+
+Returns all KIE containers for a specified {KIE_SERVER} template.
+
+.Request parameters
+[cols="25%,45%,15%,15%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`serverId`
+|ID of the {KIE_SERVER} template for which you are retrieving KIE containers
+|String
+|Required
+|===
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "container-spec": [
+    {
+      "container-id": "employeerostering_1.0.0-SNAPSHOT",
```

Comment:
@sterobin _mortgage-process_ and _employeerostering_ are product specific examples afaik, I would prefer to use the provided sample project as the reference.

Reply:
Sure. Feel free to change the example project data any way you'd like as you apply this content to the Swagger doc.